### PR TITLE
Wheel build support for Windows

### PR DESCRIPTION
**Pull request by peterjc123**

### CODE REVIEW

File: win-wheel/build_all.bat [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@setlocal EnableDelayedExpansion
+for %%v in (
+        py35
+        py36
+       ) do (
+            REM Activate Python Environment
+            set "PATH=c:\miniconda2\envs\%%v;c:\miniconda2\envs\%%v\scripts;C:\Miniconda2\envs\%%v\Library\bin;%ORIG_PATH%"
+            for %%c in (
+                cpu
+                80
```
Comment (peterjc123) [inline win-wheel/build_all.bat]: @yf225 I don't know exactly the reason that magma80 fails. Could you please look into it?

**Comment by Placinta** [comment]
Hi. Just wanted to find out, are these the commits that are used to build the official Windows wheels that are listed on https://pytorch.org/ under the Windows pip installation section?

Because I couldn't find any reference to official windows wheels neither in the jenkins logs, nor in this repo, or the main one.

**Comment by peterjc123** [comment]
Yes, it was used to build official wheels, but just a little bit out of date. If you want to build wheels by yourself, I could update this.

**Comment by Placinta** [comment]
No, it's fine, that is not a necessity for me. Thanks for replying.

**Pull request closed without merging**